src/fluent/stdout.h

```diff
@@ -46,7 +46,7 @@ class Stdout {
   }
 
  private:
-  std::set<std::tuple<std::string>> deferred_merge_;
+  std::vector<std::tuple<std::string>> deferred_merge_;
```

Comment:
I don't think that `StreamRaInto` should take in a vector, and I don't think any of the Collections' internal storage should be vectors. Bloom is all based on sets, so we should be removing duplicates and we should not be providing any ordering guarantees or anything like that.